app/views/dashboard/_latest-jobs.html.erb

```diff
@@ -0,0 +1,15 @@
+<h4 class="header"><%= link_to _("Latest Jobs"), job_invocations_path(:order=>'started_at DESC')%></h4>
+<table class="<%= table_css_classes('table-fixed') %>">
+  <tr>
+    <th class="col-md-5"><%= _("Name") %></th>
+    <th class="col-md-2"><%= _("State") %></th>
+    <th class="col-md-3"><%= _("Started") %></th>
+  </tr>
+  <% JobInvocation.latest_jobs.each do |invocation| %>
+    <tr>
+      <td class="ellipsis"><%= link_to_if_authorized invocation_description(invocation), hash_for_job_invocation_path(invocation).merge(:auth_object => invocation, :permission => :view_job_invocations, :authorizer => authorizer) %></td>
+      <td><%= link_to_invocation_task_if_authorized(invocation) %></td>
+      <td><%= time_in_words_span(invocation.start_at) %></td>
+    </tr>
+  <% end %>
```

Comment:
If there are no jobs, it just shows an empty table. Could we place a placeholder in there? Something along the lines of what we show in "Host in build mode", "latest events" or "Host configuration chart for ansible"?

Reply:
I think it would be nice to have there some placeholder. But I am not sure if we want to keep link to `/job_invocations` page when no jobs are available.